src/main/java/HttpServer/ResponseWriter.java

```diff
@@ -1,29 +1,32 @@
 package HttpServer;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
 
 public class ResponseWriter {
 
     private RequestParser parser;
     private Writable writing;
+    private ArrayList<String> resources;
 
     public ResponseWriter(RequestParser parser, Writable writing) {
-        this.parser = parser; // Intermediate class here? This has strings.
+        this.resources = new ArrayList<>();
+        resources.add("/");
+        this.parser = parser;
         this.writing = writing;
     }
 
     public void write() throws IOException {
+        HashMap request = parser.parse();
 
         System.out.println("\nWriting response:");
         System.out.println("--------------------------");
 
-        String requestLine = "HTTP/1.1 200 OK";
```

Comment:
You have a variable that is the same name as a method. Probably should try and avoid this, can become confusing for someone that will later edit your code. Ideally variable names should be nouns and methods should be verbs.

Reply:
:+1:

Reply:
:white_check_mark: